### PR TITLE
Ravinder | Development | GIDK-674 | Hide GST Composition field for UAE Company

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Ravinder | Development | GIDK-674 | Hide GST Composition field for UAE Company

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export class ContentFilterComponent implements OnInit, OnChanges, OnDestroy {
     /** True, if company country supports other tax (TCS/TDS) */
     public isTcsTdsApplicable: boolean;
     private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    public showGstComposition: boolean = false;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.ts:51]: Add JS doc

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts [hunk 2]
```diff
@@ -31,7 +31,9 @@ export class OutTemplateComponent implements OnInit, OnDestroy, OnChanges {
 	public fieldsAndVisibility: any;
 	public companyUniqueName: string;
 	public voucherType = 'default';
-	private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    public companyUniqueName$: Observable<string>;
+    public showGstComposition: boolean = false;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts:36]: Add JS Doc

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts [hunk 3]
```diff
@@ -31,7 +31,9 @@ export class OutTemplateComponent implements OnInit, OnDestroy, OnChanges {
 	public fieldsAndVisibility: any;
 	public companyUniqueName: string;
 	public voucherType = 'default';
-	private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    public companyUniqueName$: Observable<string>;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts:35]: Missing JS doc for new members

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts [hunk 4]
```diff
@@ -45,7 +47,21 @@ export class OutTemplateComponent implements OnInit, OnDestroy, OnChanges {
 			companyUniqueName = ss.companyUniqueName;
 			companies = ss.companies;
 			this.companyUniqueName = ss.companyUniqueName;
-		});
+        });
+        
+        this.companyUniqueName$ = this.store.select(p => p.session.companyUniqueName).pipe(takeUntil(this.destroyed$));
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/out.template.component.ts:52]: Avoid single character variable names

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.ts [hunk 5]
```diff
@@ -32,7 +32,7 @@ export class GstTemplateAComponent implements OnInit, OnDestroy, OnChanges {
 	@Input() public imageSignatureSrc: string;
 	@Input() public showImageSignature: boolean;
 	@Input() public templateUISectionVisibility: TemplateContentUISectionVisibility = new TemplateContentUISectionVisibility();
-
+    @Input() public showGstComposition: boolean = false;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.ts:35]: Missing JS doc for new members

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-e/gst-template-e.component.ts [hunk 6]
```diff
@@ -30,7 +30,7 @@ export class GstTemplateEComponent implements OnInit, OnDestroy, OnChanges {
     @Input() public inputTemplate: CustomTemplateResponse = new CustomTemplateResponse();
     @Input() public logoSrc: string;
     @Input() public templateUISectionVisibility: TemplateContentUISectionVisibility = new TemplateContentUISectionVisibility();
-
+    @Input() public showGstComposition: boolean = false;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-e/gst-template-e.component.ts:33]: Missing JS doc for new members

**Pull request merged**